hypothesis-python/docs/details.rst

```diff
@@ -437,12 +437,11 @@ and should be rewritten as:
 .. code:: python
 
     from unittest import TestCase
-    import inspect
 
     class TestRunTwice(TestCase):
         def execute_example(self, f):
```

Comment:
Huh. I had thought for some reason that we didn't document the executors API right now.